### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by jormello**
*Description of changes:*
Bumps the version of package dependencies for the activity resource. 

*Testing:*
- `mvn package` 
- `mvn test`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DeleteHandler.java [hunk 1]
```diff
@@ -33,9 +33,7 @@
 
             if (!currentContext.isDeletionStarted()) {
                 if (!doesStateMachineExist(model, proxy, sfnClient)) {
-                    return ProgressEvent.<ResourceModel, CallbackContext>builder()
-                            .status(OperationStatus.SUCCESS)
-                            .build();
+                    throw getStateMachineDoesNotExistException();
```
Comment (wong-a) [inline statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DeleteHandler.java:36]: Is this intentional? This is a behaviour change unrelated to dependency versions.

Comment (jormello) [reply statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DeleteHandler.java]: This change was included as part of https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-stepfunctions/pull/21. I have merged the changes from `main` into the branch, and these are no longer included as part of the file diff.

**Comment by wong-a** [comment]
> Bumps the version of package dependencies for the activity resource.

Why?

**Review by yuan-bwn — APPROVED** [review]

**Comment by jormello** [comment]
> > Bumps the version of package dependencies for the activity resource.
> 
> Why?

These dependency bumps are expected to unblock internal build of the package.

**Pull request merged**